kubernetes/resource_kubernetes_persistent_volume_test.go

```diff
@@ -880,6 +905,53 @@ resource "aws_ebs_volume" "test" {
 `, name, zone, diskName)
 }
 
+func testAccKubernetesPersistentVolumeConfig_azure_basic(name, diskName, region string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_persistent_volume" "test" {
+	metadata {
+		annotations {
+			TestAnnotationOne = "one"
+			TestAnnotationTwo = "two"
+		}
+
+		labels {
+			TestLabelOne   = "one"
+			TestLabelTwo   = "two"
+			TestLabelThree = "three"
+		}
+
+		name = "%s"
+	}
+	spec {
+		capacity {
+			storage = "2Gi"
+		}
+		access_modes = ["ReadWriteOnce"]
+		persistent_volume_source {
+			azure_disk {
+				caching_mode = "None"
+				data_disk_uri = "${azurerm_managed_disk.dbstorage.id}"
+				disk_name = "%s"
+				kind = "Managed"
+			}
+		}
+	}
+}
+resource "azurerm_managed_disk" "dbstorage" {
```

Comment:
Due to the way the testing framework is designed, you would have to explicitly include the AzureRM provider in the test setup here: https://github.com/terraform-providers/terraform-provider-kubernetes/blob/69a6932390c68a14052d1e732fce3799f78d5c5c/kubernetes/provider_test.go#L25

Use the AWS or Google entries already in there as a guideline.

Reply:
Added:
``
go get github.com/terraform-providers/terraform-provider-azurerm/azurerm
``
and modified map as requested.

Reply:
Hi Alex, is it ok ? Does this solve the issue ?